editor/editor_node.cpp

```diff
@@ -1224,7 +1224,7 @@ void EditorNode::save_resource_in_path(const Ref<Resource> &p_resource, const St
 }
 
 void EditorNode::save_resource(const Ref<Resource> &p_resource) {
-	if (p_resource->get_path().is_resource_file()) {
+	if (p_resource->get_path().is_resource_file() && !FileAccess::exists(p_resource->get_path() + ".import")) {
```

Comment:
This is an interesting addition, but would add a comment to explain what you wanted to do, like :
`// If the resource has been imported, ask the user to use a different path in order to save it`